psutil/_pslinux.py

```diff
@@ -1897,9 +1894,14 @@ def create_time(self):
         # We first divide it for clock ticks and then add uptime returning
         # seconds since the epoch.
         # Also use cached value if available.
-        bt = BOOT_TIME or boot_time()
+        bt = boot_time()
         return (ctime / CLOCK_TICKS) + bt
 
+    @wrap_exceptions
+    def create_monotonic(self):
```

Comment:
Please rename this to   'create_time_monotonic`.

Reply:
Also I think you can refactor this a bit and have `create_time` call `create_time_monotonic`.